onnx/backend/test/runner/__init__.py

```diff
@@ -15,6 +15,7 @@
 import time
 import unittest
 from collections import defaultdict
+from pathlib import Path
```

Comment:
Please make sure it is cleaned up. Otherwise lgtm.

Reply:
?

Reply:
Apologies! I was relying on the linter CI step which passed after you made this comment!

Reply:
No worries. Yeah unfortunately some aspects are not covered by the CI